### DISCUSSION TITLE
MQTT TLS und weitere MQTT Themen

### DISCUSSION THREAD

**Discussion by 1tk2** [pixelit-project/PixelIt#64] [Ideas]
Hallo,
nachdem ich nach 2 Jahren mal wieder dazu komme. mich mit dem Thema PixelIT zu beschäftigen ;-)
Vielen Dank für dies tollen Weiterentwicklung.

Ich würde gern nochmal das Themen von damals aufwärmen:
https://forum.bastelbunker.de/d/59-mqtt-tls-offener-quellcode
So wie ich das sehe, wird als MQTT Lib PubSubClient eingesetzt, da sollte es doch möglich sein TLS zu implementieren.
Vielleicht könnte man sich hier inspirieren lassen:
https://github.com/arendst/Tasmota/blob/development/tasmota/xdrv_02_9_mqtt.ino

Auch ein Thema von "damals": Sendetätigkeit MQTT - bzw. Reduzierung der Netzwerklast. Vielleicht ließen sich die Infos, welche über die Topic MatrixInfo gesendet werden, konfigurieren, sodass sich ständigen ändernde Werte nicht gesendet werden. MatrixInfo light ;-)
Auch wäre es toll wenn man das Senden der Helligkeit deaktivieren könnte.

Weiterhin wird kein LTW im MQTT generiert, um festzustellen dass PixelIT Offline ist. - Tritt das bei anderen auch auf oder liegt das an meinem MQTT Server.
Gruß

**Comment by o0shojo0o** [comment]
Hi @1tk2, 
das Thema MQTT ist für mich immer bissel schwierig da ich selbst gar kein MQTT einsetzte.
Vielleicht kann da der @foorschtbar mehr zu sagen 👀

**Reply by foorschtbar** [reply]
Joa passt alles, aber nichts was mich bisher wirklich groß gestört hätte. 

@1tk2 Mach dafür mal bitte einzelne Issues auf, dann kann man sich das mal ansehen.
